example/github.com/grpc-ecosystem/grpc-gateway/gateway_openapiv2_compile/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
-load("@rules_proto_grpc//github.com/grpc-ecosystem/grpc-gateway:defs.bzl", "gateway_swagger_compile")
+load("@rules_proto_grpc//github.com/grpc-ecosystem/grpc-gateway:defs.bzl", "gateway_openapiv2_compile")
 
-gateway_swagger_compile(
+gateway_openapiv2_compile(
```

Comment:
This is going to be a backwards incompatible change. I was likely going to have to do 3.0.0 for updated deps anyway, so should be fine